c10/util/StringUtil.cpp

```diff
@@ -146,4 +146,52 @@ size_t ReplaceAll(std::string& s, std::string_view from, std::string_view to) {
   return numReplaced;
 }
 
+template <>
+std::optional<int64_t> tryTo<int64_t>(const std::string& symbol) {
+  return tryTo<int64_t>(symbol.c_str());
+}
+
+template <>
+std::optional<int64_t> tryTo<int64_t>(const char* symbol) {
+  // TODO Using strtoll for portability. Consider using std::from_chars in the
```

Comment:
I guess this is outdated now?

Reply:
we are still using strtoll because there's no better alternative atm. When we upgrade to newer clang and msvc we should give it another try as the comments suggested here.